prodactivity/managers/RoutineManager.tsx

```diff
@@ -12,14 +13,13 @@ export default class RoutineManager {
      */
     async getRoutines(): Promise<RoutineViewModel[] | undefined> {
         try {
-            let response = await fetch(`${this.baseURL}`);
-            let jsonRoutines = await response.json();
-            
-            var routines: RoutineViewModel[] = [];
+            const jsonRoutines = await FetchHelper.get(this.baseURL);
```

Comment:
I think it would be beneficial if we could convert the json response into the actual object within Fetch helper. Maybe something like a generic function so that fetch helper would return RoutineViewModel[] instead of a json object that we have to cast as a type. 
In my backend PR I am now formatting all responses as
```
"response": response data....,
"message": "nullable error message"
```
so maybe we can wait for that to get merged then update fetch helper again (in a different PR later) to automatically handle the response and error message.